projects/elife.yaml

```diff
@@ -627,6 +626,10 @@ elife-bot:
                 "{instance}-elife-bot-sessions":
                     deletion-policy: delete
         end2end:
+            ports:
+                - 22
+                - 8020 # exposing FTP contents
```

Comment:
is this also http? (just regular http though?)

Reply:
yes, if we are accessing it through `elife.internal` it needs to be HTTP rather than HTTPS